frontend/www/js/omegaup/linkable_resource.ts

```diff
@@ -161,3 +161,55 @@ export class SchoolUser implements LinkableResource {
     return Optional.ofNonNull(`${this.getDisplayValue()}`);
   }
 }
+
+export class Contest implements LinkableResource {
+  alias: string = '';
+  title: string = '';
+
+  constructor(contest: types.Contest) {
+    this.alias = contest.alias;
+    this.title = contest.title;
+  }
+
+  toString(): string {
+    return this.title;
+  }
+
+  getUrl(): string {
+    return `/arena/${this.alias}/`;
```

Comment:
nit:

```suggestion
    return `/arena/${encodeURIComponent(this.alias)}/`;
```

no lo hacemos consistentemente en otros lugares, pero deberíamos.